src/sailfishapp_priv.cpp

```diff
@@ -79,9 +79,9 @@ QString dataDir()
     QFileInfo exe = QFileInfo(applicationPath());
 
     // "/usr/bin/<appname>" --> "/usr/share/<appname>/"
-    return QDir::cleanPath(QString("%1/%2")
-        .arg(exe.absoluteDir().filePath("../share"))
-        .arg(appName()));
+    QString path = exe.absolutePath();
+    int i = path.lastIndexOf(QChar('/')) + 1;
+    return path.replace(i, path.length() - i, "share/") + appName();
```

Comment:
Old issue, but the above comment has extra / that is not getting included in the result.